scripts/run_code_style.sh

```diff
@@ -8,7 +8,9 @@ if [ $1 == "dist_lint" ]; then
 
     # for argparse
     ls ./dist-tests/$TEMP-all-argparse/main.py
-    
+    # for python-fire
+    ls ./dist-tests/$TEMP-all-fire/main.py
```

Comment:
Should this be either `$TEMP-all-argparse/main.py` or `$TEMP-all-fire/main.py` instead of X and Y ?

Reply:
No as we testing both argparsers in one CI job so they both need to present here

Reply:
But for developers locally this will fail

Reply:
Ok this seems to work if we provide an option to choose an argparser through command line requirements